app/src/main/java/com/github/alfabravo2013/readyforexams/presentation/passwordreset/PasswordResetViewModel.kt

```diff
@@ -6,10 +6,10 @@ import androidx.lifecycle.ViewModel
 import androidx.lifecycle.viewModelScope
 import com.github.alfabravo2013.readyforexams.domain.passwordreset.PasswordResetUseCase
 import com.github.alfabravo2013.readyforexams.util.Result
-import kotlinx.coroutines.delay
 import kotlinx.coroutines.launch
 
-class PasswordResetViewModel(private val passwordResetUseCase: PasswordResetUseCase) : ViewModel() {
+class PasswordResetViewModel(
+    private val passwordResetUseCase: PasswordResetUseCase) : ViewModel() {
```

Comment:
wrong formatting here. It should be `PasswordResetViewModel(`
`private val passwordResetUseCase: PasswordResetUseCase`
`) : ViewModel() {`